python/cudnn/gemm/cutedsl/grouped/quant/api.py

```
@@ -417,6 +425,30 @@ def check_support(self) -> bool:
+        # reinterprets them. That reinterpretation is the only real override; every
+        # other format the kernel reads straight off sfa.element_type.
+
+        # e5m3 is the only override currently supported
+        self._value_error_if(
+            self.sf_fp8_dtype_override not in (None, "e5m3"),
+            f"sf_fp8_dtype_override must be None or 'e5m3', got {self.sf_fp8_dtype_override!r}",
+        )
+        if self.sf_fp8_dtype_override == "e5m3":
+            # Only allow e5m3 to pretend to be e4m3fn
+            self._value_error_if(
+                self.sf_dtype != torch.float8_e4m3fn,
```

Ruff (0.16.1)

[error] Undefined name `torch`

(F821)